api/src/main/java/org/pmiops/workbench/db/model/DbUserDataUseAgreement.java

```diff
@@ -23,7 +23,8 @@
 
   @Id
   @GeneratedValue(strategy = GenerationType.IDENTITY)
-  @Column(name = "user_data_use_agreement_id")
+  // TODO: rename this column with a Liquibase change once we're post-beta launch!
```

Comment:
Can you create a ticket for this if it doesn't already exist and link to it in the TODO?

Reply:
Good idea – done